surfman/src/platform/windows/angle/surface.rs

```diff
@@ -158,11 +159,11 @@ impl Device {
             ];
 
             EGL_FUNCTIONS.with(|egl| {
-                let egl_surface = if let Some(HandleOrTexture::Texture(texture)) = share_handle {
+                let egl_surface = if let Some(share_handle) = share_handle {
```

Comment:
These changes don't look like they compile when used with the opexnr webxr backend.

Reply:
https://github.com/jdm/surfman/commits/d3d-share is a branch that should address that.